lib/copy/__tests__/ncp/ncp-error-perm.test.js

```diff
@@ -21,7 +21,9 @@ describe('ncp / error / dest-permission', () => {
   const src = path.join(TEST_DIR, 'src')
   const dest = path.join(TEST_DIR, 'dest')
 
-  if (os.platform().indexOf('win') === 0) return
+  // when we are root, then we will be able to create the subdirectory even if
+  // we don't have the permissions to do so, so no point in running this test
+  if ((os.platform().indexOf('win') === 0) || (os.userInfo().uid === 0)) return
```

Comment:
Unneeded parenthesis
```suggestion
  if (os.platform().indexOf('win') === 0 || os.userInfo().uid === 0) return
```